pkgs/development/python-modules/unidic/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, mecab
+, setuptools-scm
+, requests
+, tqdm
+, wasabi
+, plac
+, cython
+, platformdirs
+}:
+
+buildPythonPackage rec {
+  pname = "unidic";
+  version = "1.1.0";
+  format = "setuptools";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "polm";
+    repo = "unidic-py";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-srhQDXGgoIMhYuCbyQB3kF4LrODnoOqLbjBQMvhPieY=";
+  };
+
+  postPatch = ''
+    substituteInPlace requirements.txt \
+      --replace "wasabi>=0.6.0,<1.0.0" "wasabi"
+    substituteInPlace setup.cfg \
+      --replace "wasabi>=0.6.0,<1.0.0" "wasabi"
```

Comment:
why do we need to patch both?

Reply:
My bad, I did not check whether or not both were actually needed, it looks like only the substitution for setup.cfg is required.